templates/pr-create.md.ejs

```diff
@@ -1,5 +1,53 @@
-Thanks for the pull request, @<%= payload.sender.login %>! It looks like this is your first time contributing to ESLint, so please take a moment to read over our [contribution guidelines](http://eslint.org/docs/developer-guide/contributing/pull-requests).
+<%
 
-We'll also need you to sign our [CLA](http://eslint.org/cla), which is just a way for you to say that you give us permission to use your contribution.
+var FLAG_PATTERN = /^(Fix|Update|New|Breaking|Build|Docs|Upgrade):/,
```

Comment:
Also, right now the build would still fail if only the commit message is not proper right?